lib/galaxy/model/__init__.py

```diff
@@ -687,7 +687,9 @@ class Job(JobLike, UsesCreateAndUpdateTime, Dictifiable, RepresentById):
                    FAILED='failed',
                    PAUSED='paused',
                    DELETED='deleted',
-                   DELETED_NEW='deleted_new')
+                   DELETED_NEW='deleted_new',
+                   STOP='stop',
```

Comment:
As mentioned on Gitter, STOPPING seems to me more appropriate for a job state, even if it is a request to stop the job.

Reply:
Will do. Thanks for the review!